apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/StorageClasses.cs

```diff
@@ -55,5 +55,10 @@ public static class StorageClasses
         /// and better performance/availability.
         /// </summary>
         public const string DurableReducedAvailability = "DURABLE_REDUCED_AVAILABILITY";
+
```

Comment:
Let's re-sort these as follows: Standard, Nearline, Coldine, Archive. Then Regional, Multi-regional, and Durable Reduced Availability.

The latter 3 are all considered legacy/deprecated at this point, so putting them at the bottom and adding comments to that effect (that they are accepted but not recommended for new implementations) would be great.